docs/index.md

```diff
@@ -73,16 +68,16 @@ This starts a server on port 8080.
 
 #### Running with OpenFin
 
-OpenFin has a command line tool for launching OpenFin applications. This can be installed as follows:
+The ContainerJS project provides an OpenFin-based command line tool, which can be installed as follows:
```

Comment:
@DevAndyLee I misread your comment yesterday. If we install the package globally (i.e. `npm i -g containerjs-api-openfin`) we can then use `ssf-openfin` globally. The documentation was still incorrect I think (It was saying to install `ssf-electron` directly from npm, when we need to install `containerjs-api-electron` instead). Is that correct @ColinEberhardt?